lib/Yancy/Controller/Yancy/API.pm

```diff
@@ -193,9 +193,7 @@ sub delete_item {
 # confusing to understand what the problem is
 sub _delete_null_values {
     for my $item ( @_ ) {
-        for my $key ( grep { !defined $item->{ $_ } } keys %$item ) {
-            delete $item->{ $key };
-        }
+        delete $item->{ $_ } for grep !defined $item->{ $_ }, keys %$item;
```

Comment:
This sounds like a performance enhancement. How much does it improve performance? Could you add that information to the commit message? (Why this change is being made and what impact it has?)

Reply:
I'm just rewording the message. I have not benchmarked it, but I have heard many "proper" Perl experts (unlike me) say that is one way to get better performance.

Reply:
Force-pushed with updated message.

Reply:
Here is an investigation from 2006 (https://perlmonks.org/?node_id=536503). My reasoning was that given `_delete_null_values` is such a "hot" bit of code, called for most operations, that even the chance of a bit of extra speed (and for code I find more readable, but you might disagree!) was worth it.